chia/wallet/wallet_node.py

```diff
@@ -672,7 +672,7 @@ async def receive_and_validate(inner_states: List[CoinState], inner_idx_start: i
                     ]
                     if len(valid_states) > 0:
                         async with self.wallet_state_manager.db_wrapper.lock:
-                            self.log.info(
+                            self.log.warning(
```

Comment:
it doesn't look right to make this a warning. there's nothing wrong with this, right?